library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -658,17 +658,8 @@ libspdm_return_t libspdm_receive_spdm_response(libspdm_context_t *spdm_context,
 
         /* These are the expected possible large response types.
```

Comment:
The comment needs to change now that this is a deny-list.

Reply:
done